README.md

```diff
@@ -0,0 +1,137 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="project.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+
+
+
+# 📖 [HELLO MICROVERSE PROJECT] <a name="about-project"></a>
+
+> My first Microverse Project
+
+    Hello Microverse Project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+- **HTML**
+- **CSS**
+- **LINTERS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://margarettechworld.github.io/Hello-Microverse-Project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:MargaretTechworld/Hello-Microverse-Project.git
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@MargaretTechworld](https://github.com/githubhandle)
+- Twitter: [@KojoMargaret](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/margaret-kojo-musa-759b8026a/e)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Hello Microverse text**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project ⭐️ ⭐️ ⭐️ ⭐️ ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank the Microverse Team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+

```

Comment:
- [ ] You have done exceptionally great on your read me, You just need to update a broken link before moving to the next milestone. Currently the MIT license link in your readme file leads to a 404 error page. A broken link in a readme file makes it look unprofessional, I'll suggest you create a `MIT.md` file and add the license content, You can [use this ](https://github.com/V-Blaze/gvash-planes/blob/dev/LICENSE)and then customize it for your project. :+1: :smile: 
![Screenshot from 2023-05-31 15-13-03](https://github.com/MargaretTechworld/Hello-Microverse-Project/assets/107632780/49f99c60-7473-4785-ad6c-09f307bd7b51)